src/phase0/state_transition/slot_processing.rs

```diff
@@ -1,8 +1,13 @@
 use crate::phase0::beacon_state::BeaconState;
-use crate::phase0::mainnet::Error::InvalidSlot;
-use crate::phase0::state_transition::{Context, Error};
+use crate::phase0::mainnet::Error::{InvalidEpoch, InvalidStateRoot, TransitionToPreviousSlot};
```

Comment:
style thing my preference here is to find these variants off of `Error`

and we want to use `Error` from the `state_transition` module

```suggestion
```

Reply:
fixed